homeassistant/components/nam/__init__.py

```diff
@@ -38,15 +38,27 @@ async def async_setup_entry(hass: HomeAssistant, entry: NAMConfigEntry) -> bool:
     options = ConnectionOptions(host=host, username=username, password=password)
     try:
         nam = await NettigoAirMonitor.create(websession, options)
-    except (ApiError, ClientError, ClientConnectorError, TimeoutError) as err:
```

Comment:
`TimeoutError` and `ClientConnectorError` are caught in the backend library https://github.com/bieniu/nettigo-air-monitor/blob/ab37810e975fae8d79af7e7c2d06af77ebebbb63/nettigo_air_monitor/__init__.py#L129

and ApiError is raised instead https://github.com/bieniu/nettigo-air-monitor/blob/ab37810e975fae8d79af7e7c2d06af77ebebbb63/nettigo_air_monitor/__init__.py#L195